src/app/(app)/calendar.tsx

```
@@ -0,0 +1,101 @@
+import React from 'react';
+
+import { AgendaItemRow } from '@/components/calendar/agenda-item';
+import { AgendaList } from '@/components/calendar/agenda-list';
+import { DraggableAgendaItem } from '@/components/calendar/draggable-agenda-item';
+import { DragDropProvider } from '@/components/calendar/drag-drop-provider';
+import { Button, FocusAwareStatusBar, Text, View } from '@/components/ui';
+import type { AgendaItem } from '@/types/agenda';
```

ESLint

[error] Run autofix to sort these imports!

(simple-import-sort/imports)